docs/querying/examples.md

```diff
@@ -95,3 +95,25 @@ Assuming this metric contains one time series per running instance, you could
 count the number of running instances per application like this:
 
     count by (app) (instance_cpu_time_ns)
+  
+To find out the maximum value of a metric over a period of time, for eg. last 
+6 hours, you can query like this:
+
+    max_over_time(instance_cpu_time_ns[6h])
+ 
+If you don't want your metric values to go beyond a certain threshold, you can
+clamp them to the threshold value like this:
+
+    clamp_max(instance_cpu_time_ns, 100)
+    
+You can calculate the rolling sum of value of a metric over a period of time, 
+say 6 hours like below:
+
+    sum_over_time(some_mertic_name[6h])
+    
+Lets assume we have a deduped metric `processing_time_ns` denoting the time 
```

Comment:
This isn't what this query does (plus it's a syntax error), and this sort of thing requires much more discussion than makes sense on a page like this.

Reply:
I fixed the syntax error. thanks for pointing that out.
Yeah, I know this is a complex example. But this query counts the number of 1's within the given time period.

Reply:
There's more to this than what's here, it's far too complex to include here. It's a blog post in itself.